addon/components/cf-navigation.js

```diff
@@ -6,12 +6,46 @@ import { ComponentQueryManager } from "ember-apollo-client";
 import { task } from "ember-concurrency";
 import getNavigationQuery from "ember-caluma/gql/queries/get-navigation";
 
+/**
+ * Component to display a nested form including navigation.
+ *
+ * ```hbs
+ * {{cf-navigation documentId="myDocId" section=section subSection=subSection}}
+ * ```
+ *
+ * @class CfFormComponent
+ */
 export default Component.extend(ComponentQueryManager, {
   layout,
   documentStore: service(),
 
+  /**
+   * The ID of the nested document to display the navigation for
+   * @argument {String} documentId
+   */
   documentId: null,
-  activeDocumentId: null,
+
+  /**
+   * Can be used to pass "context" information from the outside through
+   * to custom overrides.
+   *
+   * @argument {*} context
```

Comment:
Maybe we should explicitly expect an object..

Reply:
Is there any advantage for limiting the accepted types? We're just passing the context further and don't really care about it's datatype or it's content.